Source/ChilliSource/Rendering/Base/ForwardRenderPassCompiler.cpp

```diff
@@ -481,33 +488,41 @@ namespace ChilliSource
     }
     
     //------------------------------------------------------------------------------
-    std::vector<TargetRenderPassGroup> ForwardRenderPassCompiler::CompileTargetRenderPassGroups(const TaskContext& taskContext, const RenderFrame& renderFrame) noexcept
+    std::vector<TargetRenderPassGroup> ForwardRenderPassCompiler::CompileTargetRenderPassGroups(const TaskContext& taskContext, std::vector<RenderFrame>&& renderFrames) noexcept
```

Comment:
Was to enforce that the render frames should not be used after this call. Essentially transferring them into the  compiler